README.md

```diff
@@ -26,26 +26,44 @@ remotes::install_github("r-lib/lintr")
 
 ## Usage
 
+Load the library:
+
 ```R
+library(lintr)
```

Comment:
I think we should encourage keeping `search()` clean, i.e. stick with 
 `lintr::lint_dir()` etc.

Reply:
Removed all library calls.